eng/SourceBuild.props

```diff
@@ -43,6 +43,8 @@
       <InnerBuildArgs Condition="'$(ContinuousIntegrationBuild)' != ''">$(InnerBuildArgs) /p:ContinuousIntegrationBuild=$(ContinuousIntegrationBuild)</InnerBuildArgs>
       <InnerBuildArgs>$(InnerBuildArgs) /p:AdditionalRuntimeIdentifierParent=$(BaseOS)</InnerBuildArgs>
       <InnerBuildArgs Condition="'$(SourceBuildUseMonoRuntime)' == 'true'">$(InnerBuildArgs) /p:PrimaryRuntimeFlavor=Mono /p:RuntimeFlavor=Mono</InnerBuildArgs>
+      <!-- Unset NetFrameworkMinimum property, to disable netfx projects. -->
+      <InnerBuildArgs>$(InnerBuildArgs) /p:NetFrameworkMinimum=</InnerBuildArgs>
```

Comment:
This should be done in Directory.Build.props instead in order for it work with the -sb switch.

Reply:
Thanks - will fix this.

Reply:
Fixed with commit: https://github.com/dotnet/runtime/pull/83899/commits/76779f9141352e0fd2e220ac747165584747f547